### ISSUE TITLE
contrib/db/pgsql-daloradius.sql insufficent for postgresql

### ISSUE THREAD

**Issue by jonesmz** [lirantal/daloradius#169]
The sql configuration file for postgresql doesn't appear to work properly, 

Is this a known issue?

I get errors like this:

> 
> Database error
> Error Message: DB Error: unknown error
> Debug info: SET SESSION sql_mode = ''; [nativecode=ERROR: unrecognized configuration parameter "sql_mode"]
> 
> Database error
> Error Message: DB Error: no such table
> Debug info: SELECT * FROM operators_acl WHERE operator_id='1' AND file='rep_online' [nativecode=ERROR: relation "operators_acl" does not exist LINE 1: SELECT * FROM operators_acl WHERE operator_id='1' AND file='... ^]

**Comment by Garfsfield** [comment]
Hi Michael

You will need to rewrite almost all the SQL queries to work with 
PostgreSQL as well as add functions to cater for some of the mySQL 
functions such as Now().

Most people abandon going this once they realize the implications of this.

I do not know if there is a better way to get it to work.

**Comment by jonesmz** [comment]
Alright.

I'm a bit surprised about this.

Why does README.md claim that this project supports postgresql if it doesn't?

**Comment by miadz** [comment]
amazing!!!!!!

**Comment by jonesmz** [comment]
???

**Comment by MiguelVis** [comment]
According to `README.md`:
> daloRADIUS is written in PHP and JavaScript and utilizes a database abstraction layer which means that it supports many database systems, among them the popular MySQL, PostgreSQL, Sqlite, MsSQL, and many others.

I think the problem here is that the actual SQL queries used by daloRADIUS are only compatible with MySQL / MariaDB.

**Comment by raiemaille** [comment]
Hi,
I wrote a config file for postgresql in 2019 for daloradius 1.1-2
I’m working right now for an update for daloradius 1.3-0 but keep in mind that this is just a small part of the job, because there are many MANY queries to rewrite in the source code to make it to work with postgresql…
[pgsql-daloradius-1.1-2.sql.zip](https://github.com/lirantal/daloradius/files/7988223/pgsql-daloradius-1.1-2.sql.zip)

**Comment by anayrat** [comment]
I don't think this issue should have been closed, the files are not in the main repository ??
